_schemaV2.graphql

```diff
@@ -992,6 +993,48 @@ type ArtistSeries {
   artists(page: Int, size: Int): [Artist]
   description: String
   featured: Boolean!
+  filterArtworksConnection(
```

Comment:
This entire field definition gets created by that tricky `argsToSDL` business in the stitching file, yeah?

Reply:
Yep!